app/src/main/java/ch/epfl/cs311/wanderwave/model/remote/ProfileConnection.kt

```diff
@@ -26,14 +26,11 @@ class ProfileConnection(
   override val db = database
   private val coroutineScope = CoroutineScope(Dispatchers.IO + SupervisorJob())
 
-  override fun isUidExisting(spotifyUid: String, callback: (Boolean, Profile?) -> Unit) {
+  override fun isUidExisting(firebaseUid: String, callback: (Boolean, Profile?) -> Unit) {

```

Comment:
As we said that both are the same we should probably do a refactoring of it soon where we have only one uid no ? And delete the spotify UID. Of course not for this PR